src/webview/MessageList.js

```diff
@@ -355,8 +355,7 @@ const MessageList: ComponentType<OuterProps> = connect<SelectorProps, _, _>(
       allImageEmojiById: getAllImageEmojiById(state),
       auth: getAuth(state),
       debug,
-      doNotMarkMessagesAsRead:
-        !marksMessagesAsRead(props.narrow) || globalSettings.doNotMarkMessagesAsRead,
```

Comment:
We shouldn't remove the first part, `!marksMessagesAsRead(props.narrow)`. It makes it so that some views never automatically mark as read; see the comments in `marksMessagesAsRead` for why.